### PR TITLE
#67 - adding the mission statement

### PR DESCRIPTION
**Pull request by piogor**
It follows decision (action) from the recent meetings.
As stated, it will last here for reviews and comments, for one week. 
If no request for change will be made it will be merged :).

### CODE REVIEW

**Comment by piogor** [comment]
@hardion or @andygotz or @bourtemb or @david-erb-maxiv, I will be happy if at least one of you approve or request change to this PR before I merge it :-).

File: README.md [hunk 1]
```diff
@@ -2,6 +2,23 @@
...
 
+## Mission
+
+The goal of this RFC project is to provide a formal specification of the current Tango Controls system. This specification shall include:
+
+1. concepts
+2. terminology, 
+3. protocol behaviour,
+4. conventions,
+5. common APIs,
```
Comment (piogor) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,23 @@
...
+6. common utilities,
```
Comment (piogor) [inline README.md]:
```suggestion
5. common utilities,
```

File: README.md [hunk 3]
```diff
@@ -2,6 +2,23 @@
...
+
+each on a sufficient level for preparation of specification of a new vesrion of Tango Controls (Tango v10). In that respect concepts are more important than details.
```
Comment (piogor) [inline README.md]:
```suggestion
each on a sufficient level for future evolution of Tango Controls. In that respect, concepts are more important than details.
```

File: README.md [hunk 4]
```diff
@@ -2,6 +2,20 @@
 
 This repository is the home of all Tango Open Specification.
 
+## Mission
+
+The goal of this RFC project is to provide a formal specification of the current Tango Controls system. This specification shall include:
+
+1. concepts
```
Comment (piogor) [inline README.md]:
```suggestion
1. concepts,
```

File: README.md [hunk 5]
```diff
@@ -2,6 +2,20 @@
...
+## Mission
+
+The goal of this RFC project is to provide a formal specification of the current Tango Controls system. This specification shall include:
+
+1. concepts,
+2. terminology, 
+3. protocol behaviour,
+4. conventions,
+
+each on a sufficient level for future evolution of Tango Controls and/or implementation in other languages. In that respect, concepts are more important than implmentation details.
```
Comment (bourtemb) [inline README.md]:
```suggestion
each on a sufficient level for future evolution of Tango Controls and/or implementation in other languages. In that respect, concepts are more important than implementation details.
```

File: README.md [hunk 6]
```diff
@@ -2,6 +2,20 @@
 
 This repository is the home of all Tango Open Specification.
 
+## Mission
+
+The goal of this RFC project is to provide a formal specification of the current Tango Controls system. This specification shall include:
```
Comment (piogor) [inline README.md]: Add version number

Comment (piogor) [inline README.md]:
```suggestion
The goal of this RFC project is to provide a formal specification of the current (v9.3 lts) Tango Controls system. This specification shall include:
```

File: README.md [hunk 7]
```diff
@@ -2,6 +2,20 @@
 
 This repository is the home of all Tango Open Specification.
 
+## Mission
+
+The goal of this RFC project is to provide a formal specification of the current (v9.3 lts) Tango Controls system. This specification shall include:
```
Comment (piogor) [inline README.md]: @andygotz and/or @bourtemb, I have updated this with the version number according to what we talk on Monday.
Let me know if it is ok, so I can merge this PR.

Comment (andygotz) [reply README.md]: I would just specify the version as "V9 LTS"

Comment (piogor) [inline README.md]:
```suggestion
The goal of this RFC project is to provide a formal specification of the current (V9 LTS) Tango Controls system. This specification shall include:
```

Comment (piogor) [reply README.md]: @andygotz, thanks, done. Please, let me know if it is now ok.

**Pull request merged**